### PR TITLE
Dev cjw

### PR DESCRIPTION
**Pull request by Chen-Alexander**
1: Adapt according to the callback in the base module. 2: Organize config.gradle.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
+        minSdkVersion rootProject.ext.android.minSdkVersion
+        targetSdkVersion rootProject.ext.android.targetSdkVersion
         versionCode 7
         versionName "5.4.0"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
 
-        buildConfigField 'String', 'API_BASE_URL', '"https://api.agora.io"'
+//        buildConfigField 'String', 'API_BASE_URL', '"https://api.agora.io"'
+        buildConfigField 'String', 'API_BASE_URL', '"https://api-solutions-dev.sh.agoralab.co"'
```
Comment (LichKing-2234) [inline app/build.gradle:27]: 这里需要还原成线上地址

File: app/build.gradle [hunk 2]
```diff
@@ -1,19 +1,30 @@
+def safeExtGet(prop, key) {
```
Comment (LichKing-2234) [inline app/build.gradle:1]: 这里改名就叫ExtGet吧，毕竟是unsafe

Comment (Chen-Alexander) [reply app/build.gradle]: done.

File: app/build.gradle (hunk 1 above)
Comment (Chen-Alexander) [reply app/build.gradle]: done.

**Pull request closed without merging**